lib/anycable/health_server.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'webrick'
+require 'anycable/server'
+
+module Anycable
+  # Server for HTTP healthchecks
+  module HealthServer
+    class << self
+      def start
+        return if running?
+        return if Anycable.config.health_port.nil? || Anycable.config.health_port == ""
```

Comment:
Let's move this check to `Anycable::Server#start` and use `port` as an argument for `start` method. Thus we could make `HealthServer` not depend on global config at all.